netlify.toml

```diff
@@ -0,0 +1,6 @@
+# Because we use Netlify and a single page app; this fixes 404s on child pages
+
+[[redirects]]
+  from = "/*"
+  to = "/index.html"
+  status = 200
```

Comment:
Source for the fix: https://www.netlify.com/docs/redirects/

Reply:
Host to test the fix: https://5c71dcc6e6193300089087e7--crystal-archives.netlify.com/